### PR TITLE
Style question cards to be stacked

### PR DESCRIPTION
**Pull request by wbprice**
Incremental work towards 67
screen shot 2017-04-26 at 6 41 57 pm

### CODE REVIEW

**Review by kaxcode — APPROVED** [review]
LGTM

File: frontend/js/components/Frame.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Frame extends Component {
 
     render() {
         return (
-      <section className="potato">
+      <section className="frame">
```
Comment (qwo) [inline frontend/js/components/Frame.js:12]: thanks for using better variable names

**Review by qwo — COMMENTED** [review]
looks good to me.

**Review by qwo — APPROVED** [review]
looks good to me.

**Pull request merged**